### ISSUE TITLE
Missing tag (og:logo)

### ISSUE THREAD

**Issue by marcbria** [ajnyga/openGraph#7]
I think it's not mandatory but it's recommened to add the "og:logo" tag.

Syntax is as follows:
`<meta property="og:logo" content="your value" />`

This jumped to me when I tried an article page in a OG tester:
https://orcascan.com/tools/open-graph-validator?url=https%3A%2F%2Fdag.revista.uab.cat%2Farticle%2Fview%2Fv70-n3-aroca-herranz

Again, nothing dramatic but probably will be easy to fix if the plugin is updated.

**Comment by ajnyga** [comment]
yeah saw that as well, just not sure where it is used if you have the image?

**Comment by marcbria** [comment]
Humm... you are right. It didn't appear in the [OG spec](https://ogp.me/), the [RDF schema](https://ogp.me/ns/ogp.me.ttl) or in most of the [referenced manuals](https://mojodojo.io/blog/open-graph-tags/).

Looks like is a proposal of a company but as far is not included in the specification... it's safe to ignore, isn't it?
https://clearbit.com/blog/open-graph-logo

From my understanding, it will work in a similar way than the favicon, but it's hard to tell.
In the other hand, I don't see much risk in enable it, even if we don't now exactly how it will be used.

Feel free to close the issue if you think it's too uncertain.

**Comment by ajnyga** [comment]
Sure I can keep this open.
